call_server/campaign/forms.py

```diff
@@ -19,6 +19,10 @@
 from ..utils import choice_items, choice_keys, choice_values, choice_values_flat
 from ..phone_numbers import AREA_CODE_MAP
 
+def getPhoneNumberLabel(twilio_phone_number):
+    m = re.search(r'[0-9]{3}', twilio_phone_number.number)
+    return AREA_CODE_MAP[m.group(0)] + ' ' + twilio_phone_number.number
```

Comment:
You may want to use .get here instead of [] lookup, to avoid KeyErrors for potentially missing area codes. Eg, 844 isn't in your lookup table, but it's definitely a valid toll-free prefix.